Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '../../css/index.css'; // The less-compiled css for everything
+import { FiMail, FiMessageCircle, FiRepeat, FiHeart } from 'react-icons/fi';
+
+
+
+class Footer extends React.Component {  // Figured out state
+  constructor(props) {
+    super(props);
+    this.state = {
+      repostTotal: 1,
+      lovesTotal: 3,
+      black: true
+    };
+  }
+  addRepost() {
+    this.setState({
+      repostTotal: this.state.repostTotal + 1
+    });
+  };
+  addLove() {
+    this.setState({
+      lovesTotal: this.state.lovesTotal + 1
+    });
+  };
+  render() {
+    return (
+      <footer>
+        <div className="footer">
+
+          <div className="footer-icon message">
+            <a href="#comment"><FiMessageCircle /></a>
+          </div>
+
+          <div className="footer-icon repost" onClick={() => this.addRepost()}>
+            <FiRepeat /> {this.state.repostTotal}
+          </div>
+
+          <div className="footer-icon love" onClick={() => this.addLove()}>
```

Comment:
Great job figuring out onClick, and realizing you still have to use `this.` when calling a method that's declared inside a classful component.